client/src/components/Header/components/universal-nav.css

```
@@ -170,102 +170,15 @@ button.nav-link[aria-disabled='true'] {
-  .nav-lang-menu {
-    height: min(var(--lang-menu-height), calc(100vh - var(--header-height)));
-  }
-}
-
-/* main menu must be at least as tall as lang menu (when displayed)
-   unless there isn't enough view port height */
-.nav-list.display-lang-menu {
-  min-height: min(var(--lang-menu-height), calc(100vh - var(--header-height)));
-}
-
-.nav-lang-menu > button:hover {
-  background-image: none !important;
-}
-
-.nav-lang-menu .nav-link {
-  color: #fff;
-  background-color: inherit;
-  padding-top: 0;
-}
-
-.nav-lang-button {
-  display: flex;
-  justify-content: space-between;
-  background-image: none;
-}
-
-.nav-lang-button:hover svg,
-.nav-lang-button:focus {
-  fill: var(--gray-10);
-}
-
-.nav-lang-button:focus:not(:focus-visible) {
-  fill: revert;
-}
-
-.nav-lang-button[aria-expanded='true'] + .nav-lang-menu {
-  display: block;
-}
-
-.nav-lang-button svg {
-  height: 1rem;
-  width: 1rem;
-}
-
-.dark-palette .nav-lang-button:focus {
-  fill: revert;
-}
-
-.nav-lang-menu-option[aria-current='true'] {
+.nav-lang-list-option[aria-current='true'] {
   /* check mark for current language */
   background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='32' height='32' preserveAspectRatio='xMidYMid meet' viewBox='0 0 16 16'%3E%3Cg fill='white'%3E%3Cpath d='M12.736 3.97a.733.733 0 0 1 1.047 0c.286.289.29.756.01 1.05L7.88 12.01a.733.733 0 0 1-1.065.02L3.217 8.384a.757.757 0 0 1 0-1.06a.733.733 0 0 1 1.047 0l3.052 3.093l5.4-6.425a.247.247 0 0 1 .02-.022z'%3E%3C/path%3E%3C/g%3E%3C/svg%3E");
   background-size: 1.2rem;
   background-position: calc(100% - 10px) center;
   background-repeat: no-repeat;
 }
 
-.nav-lang-menu-option[aria-current='true']:focus {
-  /* check mark for current language */
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='32' height='32' preserveAspectRatio='xMidYMid meet' viewBox='0 0 16 16'%3E%3Cg fill='currentColor'%3E%3Cpath d='M12.736 3.97a.733.733 0 0 1 1.047 0c.286.289.29.756.01 1.05L7.88 12.01a.733.733 0 0 1-1.065.02L3.217 8.384a.757.757 0 0 1 0-1.06a.733.733 0 0 1 1.047 0l3.052 3.093l5.4-6.425a.247.247 0 0 1 .02-.022z'%3E%3C/path%3E%3C/g%3E%3C/svg%3E");
-}
-
-.nav-lang-menu-option[aria-current='true']:focus:not(:focus-visible) {
-  /* check mark for current language */
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='32' height='32' preserveAspectRatio='xMidYMid meet' viewBox='0 0 16 16'%3E%3Cg fill='white'%3E%3Cpath d='M12.736 3.97a.733.733 0 0 1 1.047 0c.286.289.29.756.01 1.05L7.88 12.01a.733.733 0 0 1-1.065.02L3.217 8.384a.757.757 0 0 1 0-1.06a.733.733 0 0 1 1.047 0l3.052 3.093l5.4-6.425a.247.247 0 0 1 .02-.022z'%3E%3C/path%3E%3C/g%3E%3C/svg%3E");
-}
-
-.nav-lang-menu-option[aria-current='true']:hover {
+.nav-lang-list-option[aria-current='true']:hover {
```

```suggestion
.nav-lang-list-option[aria-current='true']:hover,
.nav-lang-list-option[aria-current='true']:focus-visible {
```

Since we are using the same background style for both focus and hover we need the checkmark to look the same on both focus and hover.